docs/synapse/userguides/storm_adv_control.rstorm

```diff
@@ -206,6 +207,36 @@ set of nodes.
   }
 
 
+.. _flow-empty:
+
+Empty Block
++++++++++++
+
+An **empty block** allows you to execute the specified Storm when there are no nodes in the pipeline. If there
+are nodes in the pipeline, the associated Storm will not be executed.
+
+**Syntax:**
+
+::
+
+  empty { <storm> }
+
+**Example:**
+
+You have a function that conditionally creates a set of nodes and you wish to be warned if no nodes have been made.
+
+::
+
+    yield $makeSomeNodes()
+    empty {
+        $lib.print("No nodes created")
+    }
+
+.. NOTE::
+
+   Unlike ``init`` and ``fini`` blocks, which are run at the beginning and end of your query respectively, ``empty``
```

Comment:
I might leave out the comparison to init/fini because "at the beginning and end" sounds like a bit of a simplification of how those work (for example they can also run at the beginning/end of a loop iteration)

Reply:
Aye, took the entire note block out since it doesn't help a huge amount.